clang/test/dpct/pytorch_ATen_cuda_ns.cu

```diff
@@ -1,7 +1,7 @@
 // RUN: rm -rf %T/pytorch_ATen_cuda_ns
 // RUN: mkdir %T/pytorch_ATen_cuda_ns
 // RUN: cp %S/pytorch_ATen_cuda_ns.cu %T/pytorch_ATen_cuda_ns/
-// RUN: cp %S/user_defined_rule_pytorch.yaml %T/
+// RUN: cp -f %S/user_defined_rule_pytorch.yaml %T/
 // RUN: cp -r %S/pytorch_cuda_inc %T/

```

Comment:
I also met copy the pytorch_cuda_inc folder failed issue. How about add a subfolder for %T, and when use the extra-arg, add this subfolder.
```suggestion
// RUN: cp -r %S/pytorch_cuda_inc %T/pytorch_ATen_cuda_ns

```